2023.md

```diff
@@ -103,3 +103,4 @@
 * [Building a Resilient and Highly Available System: Ensuring Accuracy and Fault Tolerance](https://medium.com/geekculture/building-a-resilient-and-highly-available-system-ensuring-accuracy-and-fault-tolerance-365218ef7b92) by [Sachin Mamoru](https://medium.com/@sachinmamoru)
 * [Securing the Digital Frontier: The Power of Continuous Authentication](https://yoshani-malinka.medium.com/securing-the-digital-frontier-the-power-of-continuous-authentication-5fe0b1ae53d) by [Yoshani Ranaweera](https://yoshani-malinka.medium.com/)
 * [PostgreSQL With JSON Data Types](https://medium.com/@wanshikawar.99/postgresql-with-json-data-types-c4d467b3c473) by [Rajitha Wanshika](https://medium.com/@wanshikawar.99)
+* [Vulnerability Management Process: It’s More Than Just Detecting Vulnerabilities in Software!](https://medium.com/@dinukathathsara/vulnerability-management-process-its-more-than-just-detecting-vulnerabilities-in-software-55f010f473b0) by [Dinuka Thathsara](https://medium.com/@dinukathathsara)
```

Comment:
Update the blog count line 90

Reply:
ack

Reply:
Add name here as well https://github.com/wso2/iam-blogs/blob/main/README.md

Reply:
done.